akka-projection-dynamodb/src/main/scala/akka/projection/dynamodb/internal/OffsetStoreDao.scala

```diff
@@ -59,6 +63,8 @@ import software.amazon.awssdk.services.dynamodb.model.WriteRequest
     val timestampBySlicePid = AttributeValue.fromS("_")
     val managementStateBySlicePid = AttributeValue.fromS("_mgmt")
   }
+
+  final class BatchWriteFailed(val lastResponse: BatchWriteItemResponse) extends Exception
```

Comment:
Could have a list of unprocessed items here instead? So doesn't need to be unpacked in each method below.